m4opt/missions/_ultrasat.py

```diff
@@ -0,0 +1,74 @@
+import numpy as np
+from astropy import units as u
+from astropy.coordinates import SkyCoord
+from regions import RectangleSkyRegion
+from synphot import Gaussian1D, SpectralElement
+
+from .. import skygrid
+from ..constraints import (
+    EarthLimbConstraint,
+    MoonSeparationConstraint,
+    SunSeparationConstraint,
+)
+from ..models import Detector
+from ..models.background import GalacticBackground, ZodiacalBackground
+from ..orbit import TLE
+from ..utils.dynamics import Slew
+from ._core import Mission
+
+ultrasat = Mission(
+    name="ultrasat",
+    fov=RectangleSkyRegion(
+        center=SkyCoord(0 * u.deg, 0 * u.deg), width=14.28 * u.deg, height=14.28 * u.deg
+    ),
+    constraints=[
+        EarthLimbConstraint(28 * u.deg),
+        SunSeparationConstraint(46 * u.deg),
+        MoonSeparationConstraint(23 * u.deg),
+        # GalacticLatitudeConstraint(10 * u.deg),
+    ],
+    detector=Detector(
+        # Total number of pixels
+        npix=89.9e6,
+        # Pixel scale, 5.4 arcsec / pixel
+        plate_scale=(5.4 * u.arcsec) ** 2,
+        # Area of an effective aperture of is 33 cm,
+        area=np.pi * np.square(0.5 * 33 * u.cm),
+        bandpasses={
+            "NUV": SpectralElement(
+                Gaussian1D,
+                amplitude=0.25,
+                mean=2600 * u.angstrom,
+                stddev=340 * u.angstrom,
+            ),
+        },
+        # Zodiacal light, Cerenkov radiation, and Stray light dominate ULTRASAT’s background noise.
+        background=GalacticBackground() + ZodiacalBackground(),
+        # Made up to match plot
+        read_noise=6,
+        dark_noise=12 / 300 * u.Hz,
+        gain=1,
+    ),
+    # ULTRASAT will be in a geosynchronous orbit similar to GOES-17.
+    # This is the TESS TLE downloaded from Celestrak at 2024-11-15T09:15:20Z.
+    # https://celestrak.org/NORAD/elements/weather.txt
+    orbit=TLE(
+        "1 43226U 18022A   24320.05692005 -.00000082  00000+0  00000+0 0  9997",
+        "2 43226   0.0007  47.5006 0003498 198.5164  84.4417  1.00271931 24622",
+    ),
+    # Sky grid optimized for ULTRASAT's wide field of view.
+    skygrid=skygrid.healpix(200 * u.deg**2),
```

Comment:
Does Ultrasat have an established, official reference grid yet?

Reply:
Not to my knowledge at this time.

Reply:
Have you asked?

Reply:
@lpsinger ,
If it’s okay, I have updated the parameters of constraints (Earth/Sun/Moon), the slew, and the other settings. So far, there is no official grid for ULTRASAT. Maybe we could use a default grid?  
I yes, we can close it as resolved.